jobsets/default.nix

```diff
@@ -29,6 +29,20 @@ let
       path = "jobsets/cardano.nix";
       description = "Cardano SL";
     };
+    cardano-sl-staging = {
```

Comment:
This is essentially same as the `cardano-sl`, but with `nixpkgs` and `jobsets` bumped, right, @domenkozar?